### PR TITLE
*: update "change column" and "begin"

### PR DESCRIPTION
**Pull request by zimulala**
### What is changed, added or deleted? (Required)
Update "change column" and "begin".

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): pingcap/docs-cn#3304, pingcap/docs-cn#3301

### CODE REVIEW

File: sql-statements/sql-statement-change-column.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ ERROR 1105 (HY000): can't run multi schema change
 
 * Making multiple changes in a single `ALTER TABLE` statement is not currently supported.
 * Only certain types of data type changes are supported. For example, an `INTEGER` to `BIGINT` is supported, but the reverse is not possible. Changing from an integer to a string format or blob is not supported.
+* Modifying precision of type `DECIMAL` is not supported.
```
Comment (yikeke) [inline sql-statements/sql-statement-change-column.md:68]:
```suggestion
* Modifying precision of the `DECIMAL` type is not supported.
```

File: sql-statements/sql-statement-begin.md [hunk 2]
```diff
@@ -35,10 +35,12 @@ Query OK, 0 rows affected (0.01 sec)
 
 ## MySQL compatibility
 
-This statement is understood to be fully compatible with MySQL. Any compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.
+TiDB can carry the option of `PESSIMISTIC` or `OPTIMISTIC` when executing the `BEGIN` statement to indicate the type of transaction this statement opens.
```
Comment (yikeke) [inline sql-statements/sql-statement-begin.md:38]:
```suggestion
You can add the `PESSIMISTIC` or `OPTIMISTIC` option to the `BEGIN` statement in TiDB to indicate the type of transaction that this statement starts.
```

File: sql-statements/sql-statement-commit.md [hunk 3]
```diff
@@ -17,6 +17,10 @@ In the absence of a `BEGIN` or `START TRANSACTION` statement, the default behavi
 
 ![CommitStmt](/media/sqlgram/CommitStmt.png)
 
+**CompletionTypeWithinTransaction:**
+
+![CompletionTypeWithinTransaction](/media/sqlgram/CompletionTypeWithinTransaction.png)
+
 ## Examples
 
 ```sql
```
Comment (yikeke) [inline sql-statements/sql-statement-commit.md:26]: For line 42-44, see https://github.com/pingcap/docs-cn/pull/3301/files?file-filters%5B%5D=.md#r441470759

Comment (zimulala) [reply sql-statements/sql-statement-commit.md:26]: This sentence has been added in [line44](https://github.com/pingcap/docs/pull/2922/files#diff-4bc4de2792f4bfc92c0f73ec06393007R44) .

**Comment by zimulala** [comment]
PTAL @yikeke

File: sql-statements/sql-statement-commit.md (hunk 3 above)
Comment (yikeke) [reply sql-statements/sql-statement-commit.md:26]: And line 42-43 need to be deleted? @zimulala

**Comment by yikeke** [comment]
@AilinKid Hi you need to sign the CLA.

**Comment by AilinKid** [comment]
> @AilinKid Hi you need to sign the CLA.

great, done~

File: sql-statements/sql-statement-commit.md (hunk 3 above)
Comment (yikeke) [reply sql-statements/sql-statement-commit.md:26]: Problem solved. No need to delete them. Can merge now.

Comment (zimulala) [reply sql-statements/sql-statement-commit.md:26]: No need. Just like the previous sentence, PR especially changed it at the back.

**Pull request merged**